CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to 7 days to die sever manager
+You can find all the information about the server [here](https://docs.csmm.app/en/7D2D)
+## Thanks for taking the time to contribute
+The following is a set of guidelines for contributing to 7 days to die sever manager. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Links to some resources
+
+You can join the discord server to directly communicate with the developers by clicking [here](http://catalysm.net/discord).
```

Comment:
```suggestion
You can join the [Discord server](https://catalysm.net/discord) to directly communicate with the developers.
```

Instead of making a bunch of 'here' clickables, I think it's better to integrate the link into the text itself like this. Same comment for the rest of the document. Improves readability while maintaining usability of a link